<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseDifficultyDialogue</class>
 <widget class="QDialog" name="ChooseDifficultyDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test Your Might</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="buttonEasy">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>95</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Easy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonMedium">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>95</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonHard">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>95</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hard</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
